storybook/stories/Select.stories.tsx

```diff
@@ -15,7 +20,7 @@ storiesOf('Select', module)
 
       <Select
         readonly
-        handleOptionClick={action('Callback')}
+        handleOptionClick={e => clickedSelect(e)}
```

Comment:
`handleOptionClick={clickSelect}`

Reply:
The rule is you're never allowed to add this as a comment to one of my PRs